config/default/manager_auth_proxy_patch.yaml

```diff
@@ -33,3 +33,4 @@ spec:
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
         - "--leader-election-id=floorist-operator"
+        - --ansible-args='--extra-vars "floorist_image_tag=fe12542"'
```

Comment:
We should move the tag to a single source of truth somehow.

Reply:
Changed to a single file. It is applied as a jsonpatch:
https://github.com/RedHatInsights/floorist-operator/pull/2/files#diff-b97d87f831421a83db346cae215533be2ef9dc7619971db6c9868dcc37a1a6b9